reset_fmu.py

```diff
@@ -43,7 +43,7 @@ def main():
     GPIO.output(reset_pin, GPIO.LOW)
 
     # Renable vbus detect to enable usb interface
-    time.sleep(1)
+    #time.sleep(1)
```

Comment:
Are you sure we don't want a short sleep here?

Reply:
what is the purpose of a delay before driving VBUS detect high? I had to remove the delay because it was missing the bootloader, I think it only spends a couple hundred ms in the bootloader

Reply:
Ah wait I had this sleep to delay driving the vbus pin to let the flight controller boot quickly and not wait the 5 seconds in the bootloader.